src/components/app/ProfileSharing.css

```diff
@@ -110,3 +110,13 @@
   width: 14em;
   margin: 5px -5px;
 }
+
+.profileSharingShareNetworkUrlsContainer {
+  margin: 20px 0;
+}
+
+.profileSharingShareNetworkUrlsCheckbox {
+  vertical-align: middle;
+  position: relative;
+  bottom: 1px;
```

Comment:
I tried to find a better solution, using flex, or other things, but couldn't find anything really good, so let's keep this even if I'm not very happy with this "bottom" adjustment...

Reply:
I'm not very happy with the solution either but I also couldn't find a better one :(